src/Symfony/Component/Messenger/Stamp/FutureStamp.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Stamp;
+
+use Amp\Future;
+
+readonly class FutureStamp implements StampInterface
```

Comment:
```suggestion
final readonly class FutureStamp implements StampInterface
```

And some PHPDoc please.